cluster/addons/istio/noauth/istio.yaml

```diff
@@ -1,368 +1,1255 @@
-# GENERATED FILE. Use with Kubernetes 1.7+
-# TO UPDATE, modify files in install/kubernetes/templates and run install/updateVersion.sh
-#
-# Required for Istio as k8s addon.
 apiVersion: v1
 kind: Namespace
 metadata:
-  name: default
+  name: kube-public
   labels:
-    istio-injection: enabled
-    addonmanager.kubernetes.io/mode: Reconcile
```

Comment:
Turns out it is writable but regardless...
istio-injection: enabled was removed from default because the webhook matcher now looks for namespaces that do not have "istio-injection: disabled" rather than having enabled. This just covers all possible namespaces outside the special ones.
Reconcile could be added here but I wonder if there's any risk of reconciling namespace resources.